cmd/helm/init.go

```diff
@@ -284,6 +284,7 @@ func (i *initCmd) run() error {
 				}
 				fmt.Fprintln(i.out, "\nTiller (the Helm server-side component) has been upgraded to the current version.")
 			} else {
+				debug("[INFO] The error received while trying to init: %s", err)
```

Comment:
Remove the '[INFO]' label as debug function uses '[debug]' label.

Reply:
Updated